src/app/reducers/vm/redux/vm.reducers.ts

```diff
@@ -107,7 +106,8 @@ export function listReducer(state = initialListState, action: vmActions.Actions)
       };
     }
 
-    case vmActions.VIRTUAL_MACHINE_LOADED: {
+    case vmActions.VIRTUAL_MACHINE_LOADED:
+    case VmSnapshotActionTypes.RevertSuccess: {
```

Comment:
Strange and inconsistent action types naming. 

Also, you put an action related to another reducer here. I guess it shouldn't be done like that. Based on my knowledge of the redux pattern,  there should be special action related exactly for this reducer.